### ISSUE TITLE
README confusing: regarding package-name

### ISSUE THREAD

**Issue by techieshark** [google-github-actions/release-please-action#548]
### TL;DR

The README should clarify what `package-name` is for or that it should be not used verbatim. 

### Expected behavior

If the README says to copy/paste something, I expect it to work verbatim.

### Observed behavior

The README says

> Now create a .github/workflows/release-please.yml file with these contents:
```
on:
  push:
    branches:
      - main
name: release-please
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          release-type: node
          package-name: release-please-action
```

but unless I'm mistaken, users are actually expected to change that last line according to their needs (however, it's not immediately obvious what that line does or what one should change it to).

As a result (of not changing it), I run the action and see:
```
✔ Looking for latest release on branch: develop with prefix: release-please-action
```

And I'm not sure what the `with prefix: release-please-action` is… I assume it is related to `package-name: release-please-action` but it seems confusing.

### Action YAML

```yaml
on:
  push:
    branches:
      - develop
name: release-please
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          release-type: node
          package-name: release-please-action
```
```


### Log output

```text
##[debug]Starting: Set up job
Current runner version: '2.294.0'
Operating System
Virtual Environment
Virtual Environment Provisioner
GITHUB_TOKEN Permissions
Secret source: Actions
##[debug]Primary repository: my-org/my-package-name
Prepare workflow directory
##[debug]Creating pipeline directory: '/home/runner/work/my-package-name'
##[debug]Creating workspace directory: '/home/runner/work/my-package-name/my-package-name'
##[debug]Update context data
##[debug]Evaluating job-level environment variables
##[debug]Evaluating job container
##[debug]Evaluating job service containers
##[debug]Evaluating job defaults
Prepare all required actions
Getting action download info
Download action repository 'google-github-actions/release-please-action@v3' (SHA:3b2f676e018b261883c479d771eb649d27f557fe)
##[debug]Download 'https://api.github.com/repos/google-github-actions/release-please-action/tarball/3b2f676e018b261883c479d771eb649d27f557fe' to '/home/runner/work/_actions/_temp_3cdd8e90-d83c-4e91-bd2b-66561b423d9f/399f8b62-eaf6-4f07-8591-d1d4a8f9d067.tar.gz'
##[debug]Unwrap 'google-github-actions-release-please-action-3b2f676' to '/home/runner/work/_actions/google-github-actions/release-please-action/v3'
##[debug]Archive '/home/runner/work/_actions/_temp_3cdd8e90-d83c-4e91-bd2b-66561b423d9f/399f8b62-eaf6-4f07-8591-d1d4a8f9d067.tar.gz' has been unzipped into '/home/runner/work/_actions/google-github-actions/release-please-action/v3'.
##[debug]action.yml for action: '/home/runner/work/_actions/google-github-actions/release-please-action/v3/action.yml'.
##[debug]Set step '__google-github-actions_release-please-action' display name to: 'Run google-github-actions/release-please-action@v3'
##[debug]Collect running processes for tracking orphan processes.
##[debug]Finishing: Set up job


##[debug]Evaluating condition for step: 'Run google-github-actions/release-please-action@v3'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: Run google-github-actions/release-please-action@v3
##[debug]Loading inputs
##[debug]Evaluating: github.token
##[debug]Evaluating Index:
##[debug]..Evaluating github:
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'token'
##[debug]=> '***'
##[debug]Result: '***'
##[debug]Loading env
Run google-github-actions/release-please-action@v3
✔ Looking for latest release on branch: develop with prefix: release-please-action
✔ Building releases
✔ Building strategies by path

::set-output name=paths_released::[]
##[debug]='[]'
✔ Looking for latest release on branch: develop with prefix: release-please-action
✔ Building pull requests
✔ Building strategies by path
✔ Collecting release commit SHAs
✔ Collecting commits since all latest releases
✔ Splitting 1 commits by path
✔ Building candidate release pull request for path: .
✔ Considering: 0 commits
✔ No commits for path: ., skipping
##[debug]Node Action run completed with exit code 0
##[debug]Finishing: Run google-github-actions/release-please-action@v3


##[debug]Starting: Complete job
Uploading runner diagnostic logs
##[debug]Starting diagnostic file upload.
##[debug]Setting up diagnostic log folders.
##[debug]Creating diagnostic log files folder.
##[debug]Copying 1 worker diagnostic logs.
##[debug]Copying 1 runner diagnostic logs.
##[debug]Zipping diagnostic files.
##[debug]Uploading diagnostic metadata file.
##[debug]Diagnostic file upload complete.
Completed runner diagnostic log upload
Cleaning up orphan processes
##[debug]Finishing: Complete job

```
```


### Additional information

In addition to the README being a bit unclear… the thing that prompts me to go down this debugging path is the fact that no commits are being found. Release-please worked once in my repo; created a release PR and I merged that, but after that it hasn't detected any releaseable changes (despite having some that start `fix:` and `feat:` merged onto the `develop` branch after that first release was made).

**Comment by techieshark** [comment]
Possibly somewhat related: https://github.com/google-github-actions/release-please-action/issues/491

**Comment by chingor13** [comment]
#803 addressed this
